src/main/java/uk/gov/hmcts/reform/iacasenotificationsapi/infrastructure/config/NotificationGeneratorConfiguration.java

```diff
@@ -950,4 +950,23 @@
         );
     }
 
+    @Bean("submitAppealPayOfflineNotificationGenerator")
+    public List<NotificationGenerator> submitAppealPayOfflineLegalRepNotificationHandler(
```

Comment:
Possibly best for us to rename this method to `submitAppealPayOfflineNotificationHandler` seeing we're sending out notifications now to both Legal Rep and Admin Officer.